executor/simple_test.go

```diff
@@ -630,7 +630,15 @@ partition by range (a) (
 	tk.MustExec("drop stats t partition p0")
 	checkPartitionStats("global", "p1")
 
-	tk.MustExec("drop stats t partition global")
+	err := tk.ExecToErr("drop stats t partition abcde")
```

Comment:
Please add a partition which names `global` to table `t` for more tests.

Reply:
A nice comment. It is fixed.